app/models/git/exercise.rb

```diff
@@ -8,7 +8,7 @@ def initialize(exercise, git_slug, git_sha)
   end
 
   def instructions
-    lines = exercise_reader.readme.lines
+    lines = exercise_reader.readme.split("\n")
     lines.shift if /^#\s*#{@title}\s*$/.match? lines.first
     ParseMarkdown.(lines.join("\n"))
```

Comment:
I was wondering how this worked: 
`lines` doesn't remove the final newline from the strings, but `split` does.
Removing the "\n" from the join would also have fixed the problem.